### PR TITLE
User can view search results

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-04-13 at 3 54 34 PM
Screen Shot 2021-04-13 at 3 54 41 PM
Screen Shot 2021-04-13 at 3 54 55 PM

### CODE REVIEW

File: client/pages/search-results.jsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    this.state = {
+      product: null
+    };
+  }
+
+  componentDidMount() {
+    if (Number(this.props.results)) {
+      fetch(`https://app.ticketmaster.com/discovery/v2/events?apikey=AJhKApTPLUNlyEAiqkodR7pR6b9Ht1Rq&latlong=${this.props.results}&locale=*&sort=date,asc`)
+        .then(res => res.json())
+        .then(data => <RenderSearch results={data}/>)
```
Comment (thebearingedge) [inline client/pages/search-results.jsx]: This doesn't do what you think. If you need to show something, you have to `setState` and create react elements in your `render` method.

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
+    "node-zillow": "^2.0.0",
```
Comment (thebearingedge) [inline package.json]: You can `npm uninstall node-zillow`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by ameliatalkington** [comment]
Screen Shot 2021-04-13 at 4 17 45 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**